src/main/java/com/laserfiche/api/client/APIServer/SessionKeyInfo.java

```diff
@@ -0,0 +1,40 @@
+package com.laserfiche.api.client.APIServer;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+
+public class SessionKeyInfo {
+    @JsonProperty("access_token")
+    private String accessToken = null;
```

Comment:
In the C# counterpart, the class with the same name has three fields with different name. Is there any reason that we name them different here? (Also, any reason we have different number of fields?)

Reply:
the rest of the fields for the other classes i believe are following the camel case naming convention, so I thought I would keep it consistent, but i can change it to match that of the C# lib if you want. As for the number of fields in the class, i believe this class also has 3 fields with a setter and getter for each field

Reply:
Oh sorry. I was looking at an outdated version of the .Net counterpart.